### ISSUE TITLE
Not able to add Background Plugin

### ISSUE THREAD

**Issue by shanthi-techie** [react-page/react-page#1155]
Not able to add background plugin 

Error: 

Type 'CellPlugin<BackgroundState, BackgroundState>' is not assignable to type 'CellPlugin<unknown, unknown>'.

```
export const childCellPlugins: CellPlugin[] = [
    defaultSlate,
    spacer,  
    divider,  
   imagePlugin({ imageUpload: ImageUploadService('/assets/images/lowes_logo.png') }),, 
    table(),
    FooterPlugin,
    slateForFormFields,
    background({
      imageUpload: ImageUploadService('/images/default.svg'),
      enabledModes:
        ModeEnum.COLOR_MODE_FLAG |
        ModeEnum.IMAGE_MODE_FLAG |
        ModeEnum.GRADIENT_MODE_FLAG,
    }),
  ];
```

```
import React from 'react';
import type { ImageUploadType } from '@react-page/plugins-image';

const ImageUploadService: (url: string) => ImageUploadType =
(defaultUrl) => (file, reportProgress) => {
  return new Promise((resolve, reject) => {
    let counter = 0;
    const interval = setInterval(() => {
      counter++;
      reportProgress(counter * 10);
      if (counter > 9) {
        clearInterval(interval);
        resolve({ url: URL.createObjectURL(file) });
      }
    }, 100);
  });
};

export default ImageUploadService;
```

@React-page version is 4.5.0

**Comment by macrozone** [comment]
duplicate of https://github.com/react-page/react-page/issues/1104

currently, disable strict mode. i hope i can fix this asap.

**Comment by shanthi-techie** [comment]
I've already have it disabled.

```
{
  "compilerOptions": {
    /* Visit https://aka.ms/tsconfig.json to read more about this file */
    "target": "es5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
    "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
    
    "jsx": "react", /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */    
    "strict": true, /* Enable all strict type-checking options. */
    "esModuleInterop": true, /* Enables emit interoperability betwee CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */    
    /* Advanced Options */
    "skipLibCheck": true, /* Skip type checking of declaration files. */
    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
  }
}
```

**Comment by shanthi-techie** [comment]
I've already have it disabled.

```
{
  "compilerOptions": {
    /* Visit https://aka.ms/tsconfig.json to read more about this file */
    "target": "es5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
    "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
    
    "jsx": "react", /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */    
    "strict": false, /* Enable all strict type-checking options. */
    "esModuleInterop": true, /* Enables emit interoperability betwee CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */    
    /* Advanced Options */
    "skipLibCheck": true, /* Skip type checking of declaration files. */
    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
  }
}
```

**Comment by macrozone** [comment]
> I've already have it disabled.
> 
> ```
> {
>   "compilerOptions": {
>     /* Visit https://aka.ms/tsconfig.json to read more about this file */
>     "target": "es5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
>     "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
>     
>     "jsx": "react", /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */    
>     "strict": false, /* Enable all strict type-checking options. */
>     "esModuleInterop": true, /* Enables emit interoperability betwee CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */    
>     /* Advanced Options */
>     "skipLibCheck": true, /* Skip type checking of declaration files. */
>     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
>   }
> }
> ```

you sure? i just tried your snippet in the example project and it does not yield any error. Maybe you have to reload the project?

**Comment by shanthi-techie** [comment]
Ok, I will double check this from my end But i'm sure i've strict mode disabled from the beginning because i'm aware of these issues.

**Comment by shanthi-techie** [comment]
It worked only after upgrading the version to 4.5.3
